src/api/useApiClient.ts

```diff
@@ -0,0 +1,7 @@
+import { useGlobalContext } from '../common/context/useGlobalContext';
+import { getApiClient } from './getApiClient';
+
+export function useApiClient() {
+  const baseUrl = useGlobalContext().activeNetworkKey;
```

Comment:
Maybe we should put this in the redux store so we don't have to create this object however many times